### PR TITLE
Inverted prop for reverse messages display order

### PR DESCRIPTION
**Pull request by apparition47**
Amendment to the #489 PR allow the user to set a prop to reverse the `messages` display order on the scroll view.

 **`inverted`** _(Bool)_ - Reverses display order of `messages`; default is `true`

### CODE REVIEW

**Comment by xcarpentier** [comment]
@brunocascio why you don't merge this one?

File: src/GiftedChat.js [hunk 1]
```diff
@@ -84,14 +84,14 @@ class GiftedChat extends React.Component {
     if (!Array.isArray(messages)) {
       messages = [messages];
     }
-    return currentMessages.concat(messages);
+    return this.inverted ? messages.concat(currentMessages) : currentMessages.concat(messages);
```
Comment (xcarpentier) [inline src/GiftedChat.js]: Should be `this.props.inverted`
Please fix.

**Comment by xcarpentier** [comment]
@brunocascio I've made the change in PR: `this.inverted => this.props.inverted`

**Comment by brunocascio** [comment]
Hi @xcarpentier ! Thanks for the PR. Merging

**Comment by xcarpentier** [comment]
This is not good at all, I hope that works when I fixed but I the original PR is not form me but from @apparition47 ... and now in my test in master it's not so good :/

**Comment by apparition47** [comment]
@xcarpentier looks like you went ahead and added those changes [here](https://github.com/FaridSafi/react-native-gifted-chat/pull/658/commits/528aa0e6abc1d17b1500203003e02c9520d830cb)

**Pull request merged**